xml/rmt_rmt-cli.xml

```diff
@@ -327,6 +327,39 @@
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli systems purge</command></term>
+    <listitem>
+     <para>
+      This command lists and optionally deletes inactive systems.
+      It has the following options:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        <option>--before <replaceable>DATE</replaceable></option> - lists systems that have been
+        inactive since <replaceable>DATE</replaceable> until now. Default is the last 3 months.
```

Comment:
```suggestion
        <option>--before <replaceable>DATE</replaceable></option>&mdash;lists 
        systems that have been inactive since <replaceable>DATE</replaceable>
        until now. Default is the last 3 months.
```